src/slacko.ml

```diff
@@ -366,6 +366,8 @@ type attachment_obj = {
   mrkdwn_in: string list option [@default None];
 } [@@deriving yojson { strict = false }]
 
+let ifnone a b = match a with Some v -> Some v | None -> b
```

Comment:
I'd name this `if_none` to match OCaml naming scheme. And maybe make this multi-line, since there is not much use in saving lines here.